### PR TITLE
Add bzlmod module file

### PR DESCRIPTION
**Pull request by hanneskaeufler**
Motivation:
https://github.com/bazelbuild/bazel-central-registry/issues/124

Open questions:
* Is the googletest project willing to maintain the bzlmod file?

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module(
+    name = "googletest",
+    compatibility_level = 1,
+    version = "0.0.0",
```
Comment (hanneskaeufler) [inline MODULE.bazel:4]: In order not to have to maintain a version number multiple times (or at all), it seems to be a somewhat often used practice to have `0.0.0` here and just increment the version number in the bazel central registry.

**Comment by hanneskaeufler** [comment]
@derekmauro Anything I can do to move this forward? thanks!

**Comment by Vertexwahn** [comment]
@derekmauro Can we bring this forward?

File: MODULE.bazel (hunk 1 above)
Comment (Vertexwahn) [reply MODULE.bazel:4]: The version attribute is optional - maybe leave it out, instead of using 0.0.0

**Comment by Vertexwahn** [comment]
@derekmauro Please review!

**Review by derekmauro — COMMENTED** [review]
I'm not really sure how this is supposed to work.

We prefer to live at head. This means, for example, that sometimes we might use a feature in the latest head version of Abseil that isn't part of any release. How is bzlmod supposed to deal with this?

**Comment by Vertexwahn** [comment]
@derekmauro Good point! There is  `archive_override` to override a specific version with a new one: https://github.com/google/googletest/pull/4368

**Comment by hanneskaeufler** [comment]
Closing in favor of https://github.com/google/googletest/pull/4368, thanks for picking this up Vertexwahn!

**Pull request closed without merging**